knowledge-base/actions/create-release/action-security.yml

```diff
@@ -0,0 +1,5 @@
+name: 'Create a Release'
+github-token:
```

Comment:
@Devils-Knight there is no info of where the token should be. Will this work? If not, please add a test case for this. Token location must be either an input or env variable.

Reply:
@varunsh-coder I have updated the permissions for `create-release` and ` checkmarx` please verify the changes.